main/lsp/BUILD

```diff
@@ -9,25 +9,26 @@ cc_library(
         "watchman/*.cc",
     ]) + [
         ":lsp_messages",
-        "LSPFileUpdates.h",
-        "DefLocSaver.h",
-        "watchman/WatchmanProcess.h",
         "AbstractRenamer.h",
-        "NextMethodFinder.h",
-        "LocalVarFinder.h",
+        "DefLocSaver.h",
+        "ErrorFlusherLSP.h",
+        "ErrorReporter.h",
         "FieldFinder.h",
-        "json_enums.h",
-        "LocalVarSaver.h",
-        "ShowOperation.h",
-        "lsp_messages_gen_helpers.h",
+        "LSPFileUpdates.h",
         "LSPIndexer.h",
+        "LSPPathType.h",
```

Comment:
It's a bit messy here, but I did two things:
1) Added `LSPPathType.h`
2) Sort imports alphabetically